vuepress/docs/next/docs/reference/local-tips-and-tricks.md

```diff
@@ -201,7 +197,7 @@ python3 -m http.server 8000
 13. You should now be able to install Entando using the static IP. If your Entando installation stalled during startup, and was previously configured with a static IP, it should continue initializing as soon as the external address is functional again. 
 
 #### Option 4: Reinstall Entando
-We're including this option because it works and requires no additional configuration. If you plan to regularly work with Entando we recommend developing against a centralized and shared Kubernetes instance rather than running a full stack locally. If you require a local cluster we recommend using option 1 or 2.
+We're including this option because it works and requires no additional configuration. If you work with Entando regularly, we recommend developing in a centralized and shared Kubernetes instance rather than running a full stack locally. If you require a local cluster, we recommend using option 1 or 2.
```

Comment:
@nshaw can you comment on the use of "against" vs "in" here?

Reply:
@Lyd1aCla1r3 I think "in" is better.

Reply:
@nshaw i was wondering what it meant to develop against an environment... is that common phrasing?

Reply:
@Lyd1aCla1r3 it's not that unusual. The only comparison I can think of is when you develop "against" or "towards" a standard, basically with a goal of meeting a target. I do think "in" is a bit better but just personal opinion/preference on this one. We log in to or  shell/ssh into a VM, we log on to a cloud provider, we deploy into Entando, pods run in Kubernetes...